src/main/java/net/smartcosmos/usermanagement/user/service/CreateUserServiceDefault.java

```diff
@@ -22,7 +22,7 @@
 import static net.smartcosmos.usermanagement.event.UserEventType.USER_CREATE_FAILED_ALREADY_EXISTS;
 
 /**
- * Initially created by SMART COSMOS Team on July 01, 2016.
+ * The default implementation of of the {@link CreateUserService}.
```

Comment:
<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>R</kbd> in IntelliJ is amazing, especially when it adds the same typos etc. everywhere. :/